crates/astria-sequencer/src/authority/storage/values.rs

```diff
@@ -0,0 +1,149 @@
+use std::{
+    borrow::Cow,
+    fmt::{
+        self,
+        Display,
+        Formatter,
+    },
+};
+
+use astria_core::{
+    crypto::VerificationKey as DomainVerificationKey,
+    primitive::v1::ADDRESS_LEN,
+    protocol::transaction::v1alpha1::action::ValidatorUpdate as DomainValidatorUpdate,
+};
+use astria_eyre::eyre::bail;
+use borsh::{
+    BorshDeserialize,
+    BorshSerialize,
+};
+use telemetry::display::base64;
+
+use crate::{
+    accounts::AddressBytes as DomainAddressBytes,
+    authority::ValidatorSet as DomainValidatorSet,
+};
+
+#[derive(Debug, BorshSerialize, BorshDeserialize)]
+pub(crate) enum Value<'a> {
+    AddressBytes(AddressBytes<'a>),
+    ValidatorSet(ValidatorSet<'a>),
+}
+
+impl<'a> Display for Value<'a> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        match self {
+            Value::AddressBytes(address) => {
+                write!(f, "address {}", base64(address.0.as_ref()))
+            }
+            Value::ValidatorSet(_validator_set) => write!(f, "validator set"),
```

Comment:
Some `Display` impls are relatively verbose, while this one is very sparse. Seems inconsistent? Let's make this apparent by using doing `validator set {...}`?

Reply:
Fixed in [26e3683](https://github.com/astriaorg/astria/pull/1492/commits/26e36839ad833bd65b420e6f080b14b20cb4acee).